.gitignore

```diff
@@ -4,7 +4,7 @@
 /dist/
 /static/
 /.dioxus/
-
+./docs/*
```

Comment:
docs directory is used to host this web app to github pages.
do not git ignore this.